models/role.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+module.exports = (sequelize, DataTypes) => {
+  const Role = sequelize.define(
+    'role',
+    {
+      name: DataTypes.STRING,
+      displayName: DataTypes.STRING,
+    },
+    {
+      timestamps: false,
```

Comment:
good catch - normally we would want this, but for this `roles` table this is not necessary and also we don't actually want users to be able to edit entries in it, so i'm basically saying this is good lol